build-tools/heroku/tailwind/styles.css

```diff
@@ -55,6 +55,41 @@ strong {
   @apply bg-gray-500 border-gray-700;
 }
 
+
+.tooltip {
+  position: relative;
+  display: inline-block;
+ 
+}
+
+.tooltip .tooltiptext{
+    @apply invisible absolute;
+    @apply bg-white text-red-600 items-center;
+    width: 130;
+    bottom: 150%;
+    left: 50%;
+    margin-left: -80px;
+ 
+ 
+}
+.tooltip:hover .tooltiptext {
+  visibility: visible;
+}
+
+.tooltip .tooltiptext::after {
+  @apply absolute;
+  content: " ";
+  top: 100%;
+  /* At the bottom of the tooltip */
+  left: 50%;
+  margin-left: -10px;
+  border-width: 10px;
+  border-style: solid;
+  border-color: white transparent transparent transparent;
```

Comment:
Could we create a separate issue to replace this by using Tailwind classes? It keeps our CSS code clean and read-able and I'm afraid we will forget as this PR is already merged.

Reply:
to clarify, when you say tailwind classes, you mean like "@apply ...." instead of "margin-left:...", right? Do all of these have classes in tailwind? I tried researching about the "width", "bottom", "left" styles and couldn't find a class in tailwind that did the exact same thing as this does, but I can definitely create a new issue and look again if you know that all of this can be written using tailwind? @TiBiBa

Reply:
by the way, here is the issue #3783